sites/kit.svelte.dev/src/lib/search/Search.svelte

```diff
@@ -2,6 +2,8 @@
 	import { browser } from '$app/environment';
 	import { searching, query } from './stores.js';
 
+	const has_search_support = !browser || ('serviceWorker' in navigator);
```

Comment:
```suggestion
	const has_search_support = !browser || ('Worker' in window);
```

Shouldn't this check if WebWorkers are supported, instead of ServiceWorkers?
caniuse.com says they are available since IE10, ServiceWorkers since Edge17.
Same in SearchBox.svelte

Reply:
Thank you for the review!

You're totally write, it was just my poor understanding on Web Workers. Will be fixed now!

Reply:
OK, changed my mine.
You are conceptually right, but it seems by my quick testing that Vite for now doesn't handle transforming web workers well on legacy...
So I'll just add a check for 'Worker' avilability, together with the check we're not on legacy mode, i.e. `(!import.meta.env.LEGACY && 'Worker' in window)`.

Reply:
I agree.
But I can not mark this as resolved.

Reply:
> I agree. But I can not mark this as resolved.

Blocker Vite issue (I had just created): vitejs/vite#10383